commands/import_installation_test.go

```diff
@@ -265,7 +266,7 @@ var _ = Describe("ImportInstallation", func() {
 			Expect(fmt.Sprintf(format, v...)).To(Equal("finished import"))
 		})
 
-		It("it only retries 3 times before giving up", func(done Done) {
+		It("it only retries 3 times before giving up", func(ctx context.Context) {
```

Comment:
Is the Context in the test function necessary? It doesn't seem to be used.

Reply:
explained in the [previous](https://github.com/pivotal-cf/om/pull/661#discussion_r1698970284) comment